docs/DeveloperGuide.md

```diff
@@ -277,6 +277,27 @@ You can navigate the gradle terminal by clicking on elephant icon _(Gradle)_ > t
 
 2. _{ more test cases …​ }_
 
+### Adding a transaction
+
+1. Adding a transaction while all transactions are being shown with minimally 1 person in address book
+   
+   1. Prerequisites: List all persons and transactions using the `list` and `listTxn` command. At least one person in the address book list and multiple transactions in the list
+
+   2. Test cases: `addTxn 1 amt/12.3 desc/John owes me for dinner`<br>
+      Expected: A new transaction related to the person in the first index of the address book is added to the list along with description of it. The amount reflected in the transaction is displayed green in the transaction panel to signify that you are owed. The date of the transaction displays the current day. No categories to be displayed. Details of the added transaction is shown in the status message.
+
+   3. Test cases: `addTxn 1 amt/-12.3 desc/I owe John for dinner date/10102024`<br>
+      Expected: A new transaction related to the person in the first index of the address book is added to the list along with description of it. The amount reflected in the transaction is displayed red in the transaction panel to signify that I owe. The date of the transaction displays `10 Oct 2024`. No categories to be displayed. Details of the added transaction is shown in the status message.
+
+   4. Test cases: `addTxn 1 amt/12.3 desc/John owes me for dinner cat/FOOD`<br>
+      Expected: A new transaction related to the person in the first index of the address book is added to the list along with description of it. The amount reflected in the transaction is displayed green in the transaction panel to signify that you are owed. The date of the transaction displays the current day. Category of `FOOD` is displayed. Details of the added transaction is shown in the status message.
+
+   5. Test cases: `addTxn 0 amt/ desc/ date/ cat/`<br>
+      Expected: No transaction is added. Error details shown in the status message. Status bar remains the same.
```

Comment:
@GavinSin Do you know what this status bar refers to? Does the original UI have a status bar?

Reply:
Oh the statusbar is the one at the footer